### PR TITLE
AppLinks: divider

### PR DESCRIPTION
**Pull request by sorja**

### CODE REVIEW

File: webapp/app/components/header/components/AppLinks.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ const AppLinks = ({ i18n }) => {
       className="app-header__menu-item">
       FRA Platform
     </NavLink>
+    <div className="nav__divider"/>
```
Comment (minotogna) [inline webapp/app/components/header/components/AppLinks.js]: This should be visible only to when the menu after is visible

File: webapp/app/components/header/components/AppLinks.js [hunk 2]
```diff
@@ -20,6 +20,7 @@ const AppLinks = ({ i18n }) => {
       className="app-header__menu-item">
       FRA Platform
```
Comment (minotogna) [inline webapp/app/components/header/components/AppLinks.js]: Use i18n here

File: webapp/app/components/header/components/AppLinks.js (hunk 1 above)
Comment (sorja) [reply webapp/app/components/header/components/AppLinks.js]: Can you explain?

Comment (sorja) [reply webapp/app/components/header/components/AppLinks.js]: Currently looks like:
image

Comment (minotogna) [reply webapp/app/components/header/components/AppLinks.js]: Is the divider visible in statisticalfactsheets view? It should not

**Pull request merged**